playbook-website/app/views/pages/visual_guidelines_partials/_display.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= pb_rails("title", props: { text: "Display", size: 1, margin_top: "xl", padding_top: "xl" }) %>
+<%= pb_rails("body", props: { tag: "p", margin_top: "xs", margin_bottom: "lg" }) do %>
+<% end %>
```

Comment:
Empty block between do and end. Did you wanted to include text or?

Reply:
Eventually there will be a description added here once thats written up! This will also keep the spacing constant until then :)

Reply:
Interesting. You can also add a text prop to the body kit and then you don't need a block.

Reply:
Ahh great idea. I'll take a look at some of the other sections to see how they are handling this. Thanks for pointing this out!

Reply:
Of course. We just updated a lot of old pages in Nitro to meet the UX standards and I run into this a lot.